<launch>
  <node 
    pkg="mvs_ros_pkg"
    type="grabImg"
    name="mvs_camera"
    args="$(find mvs_ros_pkg)/config/left_camera.yaml"
    respawn="true"
    output="screen"
  />
  <node 
    pkg="rviz"
    type="rviz"
    name="rviz"
    args="-d $(find mvs_ros_pkg)/rviz_cfg/mvs_camera.rviz"
  />
</launch>
